src/mpid/ch4/shm/posix/posix_impl.h

```diff
@@ -4,7 +4,7 @@
  *      See COPYRIGHT in top-level directory.
  *
  *  Portions of this code were written by Intel Corporation.
- *  Copyright (C) 2011-2016 Intel Corporation.  Intel provides this material
+ *  Copyright (C) 2011-2017 Intel Corporation.  Intel provides this material
  *  to Argonne National Laboratory subject to Software Grant and Corporate
```

Comment:
Should we put copyright change in this commit ?

Reply:
I think it makes sense to update them at the same time.